cmd/vela-email/plugin_test.go

```diff
@@ -279,6 +301,20 @@ func TestValidateErrors(t *testing.T) {
 			},
 			wantErr: ErrorMissingSMTPParam,
 		},
+		{
+			name: "SMTP auth supplied but credentials missing",
+			parameters: Plugin{
+				Auth: "LoginAuth",
+				SMTPHost: &SMTPHost{
+					Host: "smtphost.com",
+					Port: "234234",
```

Comment:
Also LOL this works with an invalid port, that may be a problem if something else doesn't check that